src/views/starter-projects/starter-projects.json

```diff
@@ -5,31 +5,31 @@
    "type": "project",
    "title": "Dance Party",
    "thumbnailUrl": "https://cdn2.scratch.mit.edu/get_image/project/10128067_144x108.png",
-   "creator": "",
+   "creator": "Scratchteam",
```

Comment:
Sorry, this isn't going to work. The `Thumbnail` component in Carousel is looking for certain properties in the current `item`: 
https://github.com/LLK/scratch-www/blob/9f2ca8428339f660db87f6735360c2f1de457fb8/src/components/carousel/carousel.jsx#L70-L81
On line 71, it is looking for `item.author.username`, so the item must have a property `author`, that consists of an object that includes a property `username`

Note: the `carousel.json` example json corresponds to the data needed for the `LegacyCarousel`, not `Carousel`. Other changes include the `stats` object, and the image should be `item.image`, not `item.thumbnailUrl`